bigbluebutton-html5/imports/ui/components/video-dock/component.jsx

```diff
@@ -587,8 +588,9 @@ class VideoDock extends Component {
     const { intl, userId } = this.props;
 
     if (message.cameraId == userId) {
-      this.unshareWebcam();
       this.notifyError(intl.formatMessage(intlMessages.sharingError));
+      this.unshareWebcam();
+      this.stop(userId);
```

Comment:
Please bring `this.stop(message.cameraId)` outside of the condition block as it's currently present in both `if` and `else`